todo_app/app.py

```diff
@@ -1,18 +1,53 @@
-from flask import Flask, render_template, request
-
+from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
-import requests  
+import requests
 import os
 from todo_app.data.item import Item
 import todo_app.data.mongo_items_collect as mongo_items_collect
 from todo_app.data.view_model import ViewModel
-
+from flask_login import LoginManager, login_required, UserMixin, login_user, current_user
+import flask
+from functools import wraps
+
+class User(UserMixin):
+        def __init__(self, id):
+            role = ["read","write"]
```

Comment:
It looks like you create this variable then immediately ignore it - is it there as a pointer to future developers, or just seem now outdated code?